waiter/integration/waiter/websocket_integration_test.clj

```diff
@@ -97,7 +97,10 @@
       (is (str/includes? (.getMessage error) "Unexpected HTTP Response Status Code: 403 Forbidden"))
       (is (not (realized? connect-success-promise))))))
 
-(deftest ^:parallel ^:integration-fast test-request-auth-success
+(defn- request-auth-success-helper
+  "Helper function that establishes a websocket connection to the given url path and including
+  the given headers. Then verifies that the responses match expectations."
+  [url-path additional-headers]
```

Comment:
Maybe pass waiter-url as an argument here and move the `testing-using-waiter-url` macro to the caller?